src/renderer/widgets/WalletDetails/ui/WalletConnectDetails.tsx

```diff
@@ -14,9 +14,18 @@ import { wcDetailsModel } from '../model/wc-details-model';
 import { wcDetailsUtils, walletDetailsUtils } from '../lib/utils';
 import { ForgetStep } from '../lib/constants';
 import { Animation } from '@shared/ui/Animation/Animation';
-import { BaseModal, Button, ConfirmModal, FootnoteText, Icon, SmallTitleText, StatusModal } from '@shared/ui';
-import { GeneralWalletActions } from './WalletActions/GeneralWalletActions';
+import {
+  BaseModal,
+  Button,
+  ConfirmModal,
+  DropdownIconButton,
+  FootnoteText,
+  Icon,
+  SmallTitleText,
+  StatusModal,
+} from '@shared/ui';
 import { IconNames } from '@shared/ui/Icon/data';
+import { RenameWalletModal } from './WalletActions/RenameWalletModal';
```

Comment:
Rename / Forget and other actions are features
Please move `RenameWallet` to `features/wallet`